src/cockpit/389-console/src/lib/server/accessLog.jsx

```diff
@@ -685,6 +699,21 @@ export class ServerAccessLog extends React.Component {
                                     />
                                 </GridItem>
                             </Grid>
+                            <Grid title="Compress (gzip) the log after it's rotated.">
+                                <GridItem className="ds-label" span={3}>
+                                    Compress Rotated Logs
+                                </GridItem>
+                                <GridItem span={1}>
+                                    <Checkbox
```

Comment:
It's a subjective thing but I think 'Switch' will look better here.
Now it looks very small and "stealthy" compared to other elements on the page.

![image](https://user-images.githubusercontent.com/5634892/167751052-0d4bc8c8-e7c8-4448-9ea6-7b529b15ed2b.png)

The `Switch` will make it more visible and nice for perception.
Still, it's just my opinion and I leave the final decision fully to you.

Reply:
I switched to the Switch.  I had thought about it, but never got around to trying it out.  It is definitely better!  Thanks!

All changes are made, please review...